app/pages/collections.tsx

```diff
@@ -38,6 +38,86 @@ const CollectionsPage: BlitzPage = () => {
         refetchFn={refetch}
       />
       <main className="w-full p-8">
+        {session.workspaceId && draftCollections.length > 0 && (
+          <>
+            <h2 className="mb-4 text-center text-3xl font-extrabold">Your Collections</h2>
+            <div className="collections mx-auto grid grid-cols-2 grid-cols-3 gap-x-8 gap-y-12 md:grid-cols-4 lg:grid-cols-6">
+              {draftCollections.map((collection, index) => {
+                return (
+                  <Link key={collection.suffix} href={`/collections/${collection.suffix}/admin`}>
+                    <div
+                      className={`${
+                        collection.type.type === "COMMUNITY"
+                          ? `col-span-3 row-span-4 bg-indigo-200 text-2xl dark:bg-indigo-600 lg:row-span-2 collection-${
+                              collection.submissions.length > 4
+                                ? "4"
+                                : collection.submissions.length
+                            }`
+                          : collection.type.type === "COLLABORATIVE"
+                          ? `col-span-2 row-span-2 bg-teal-200 text-lg dark:bg-teal-600 collection-${
+                              collection.submissions.length > 4
+                                ? "4"
+                                : collection.submissions.length
+                            }-teal`
+                          : `col-span-1 bg-amber-200 text-base dark:bg-amber-600 md:col-span-1 collection-${
+                              collection.submissions.length > 4
+                                ? "4"
+                                : collection.submissions.length
+                            }-amber`
+                      }  cursor-pointer rounded-md bg-cover bg-center`}
+                      key={`${collection.title}-${index}`}
+                    >
```

Comment:
I noticed that some of these are repeated later (under all collections), which makes a good case for refactoring. No need to resolve this on this PR—I just wanted to leave a comment 👍

Reply:
💯%

sloppy of me 😅 will look into refactoring this when I update the filiename

Reply:
No worries! And, if you want me to take over refactoring, I'm happy to do so in another issue–PR 👍